libs/blockscout-service-launcher/src/database.rs

```diff
@@ -162,3 +141,113 @@ impl DatabaseKvConnection {
         )
     }
 }
+
+#[serde_as]
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+#[serde(default, deny_unknown_fields)]
+pub struct DatabaseConnectOptionsSettings {
+    /// Maximum number of connections for a pool
+    pub max_connections: Option<u32>,
+    /// Minimum number of connections for a pool
+    pub min_connections: Option<u32>,
+    /// The connection timeout for a packet connection
+    pub connect_timeout: Option<Duration>,
+    /// Maximum idle time for a particular connection to prevent
+    /// network resource exhaustion
+    pub idle_timeout: Option<Duration>,
+    /// Set the maximum amount of time to spend waiting for acquiring a connection
+    pub acquire_timeout: Option<Duration>,
+    /// Set the maximum lifetime of individual connections
+    pub max_lifetime: Option<Duration>,
+    /// Enable SQLx statement logging
+    pub sqlx_logging: bool,
+    /// SQLx statement logging level (ignored if `sqlx_logging` is false)
+    pub sqlx_logging_level: LevelFilter,
+    #[cfg(feature = "database-1_0")]
+    /// SQLx slow statements logging level (ignored if `sqlx_logging` is false)
+    pub sqlx_slow_statements_logging_level: LevelFilter,
```

Comment:
Cannot we use the tracing::log::LevelFilter directly here and not create our own enum?

Reply:
There was a deserialization problem, but now I figured we can fix it using custom serde serializers, so custom enum is no longer needed indeed.